test/test_main.py

```diff
@@ -109,25 +105,29 @@ def test_definition_syntax_error(self, data_dir):
         path = os.path.join(data_dir, 'definition_files/bad.yml')
 
         with pytest.raises(DefinitionError) as error:
-            AnsibleBuilder(filename=path)
+            AnsibleBuilder(filename=path, base_image='my-custom-image')
```

Comment:
these test changes shouldn't be needed, should they? I'd like to see if we can roll some of those back.